frontend/packages/ceph-storage-plugin/src/utils/noobaa-utils.ts

```diff
@@ -64,6 +65,16 @@ export const getBackingStoreType = (bs: BackingStoreKind): BC_PROVIDERS => {
   return type;
 };
 
+export const getNamespaceStoreType = (ns: NamespaceStoreKind): BC_PROVIDERS => {
+  let type: BC_PROVIDERS = null;
+  Object.entries(NS_PROVIDERS_NOOBAA_MAP).forEach(([v, k]) => {
```

Comment:
```suggestion
  Object.entries(NS_PROVIDERS_NOOBAA_MAP).forEach(([k, v]) => {
```
nit: The order is key, value

Reply:
fixed